package.json

```diff
@@ -6,6 +6,7 @@
     "lint": "prettier . --check --ignore-unknown",
     "fix": "prettier . --check --write --ignore-unknown",
     "generate-docs": "node --loader ts-node/esm --no-warnings --experimental-json-modules ./utils/generate-docs/index.ts",
+    "generate-release-docs": "node --loader ts-node/esm --no-warnings --experimental-json-modules ./utils/schema-utils/GenerateReleaseDocs.ts generate-release-docs -v",
```

Comment:
@xudiera I fixed an issue here by removing the `--tag main` from this. I couldn't run the script as desired via `npm run generate-release-docs -- --tag 1.0.1` for example; it would give me refs with `.../v/main,1.0.1/...` in them.

Reply:
This might imply some needed changes to how we've configured yargs and the `--tag` argument as it shouldn't support multiples, but we can proceed with this.